ci/travis/build-docker-images.py

```diff
@@ -98,7 +98,8 @@ def _get_wheel_name(minor_version_number):
         ]
         assert len(matches) == 1, (
             f"Found ({len(matches)}) matches for 'ray-*{PYTHON_WHL_VERSION}"
-            f"{minor_version_number}*-manylinux*' instead of 1")
+            f"{minor_version_number}*-manylinux*' instead of 1."
+            f"wheel matches: {matches}")
```

Comment:
Nit: So it's not smushed onto one line.
```suggestion
            f"{minor_version_number}*-manylinux*' instead of 1.\n"
            f"wheel matches: {matches}")
```